src/chart/model/scaling/viewport.model.ts

```diff
@@ -403,6 +407,14 @@ export abstract class ViewportModel extends ChartBaseElement implements Viewable
 		this._zoomY = value;
 	}
 
+	get zoomXY(): ZoomXToZoomYRatio {
```

Comment:
Hm, but the same could apply to zoomX and zoomY, it can be calculated with xStart/xEnd and yStart/yEnd, the reason why I think it should be here because it's convenient :)

Reply:
Or I misunderstoon and zoomXY should be a part of state, but without getter?